test/ASTGen/diagnostics.swift

```diff
@@ -1,14 +1,5 @@
 // RUN: %empty-directory(%t)
 
-// RUN: not %target-swift-frontend %s -dump-parse -disable-availability-checking -enable-bare-slash-regex -enable-experimental-feature ParserASTGen > %t/astgen.ast.raw
-// RUN: not %target-swift-frontend %s -dump-parse -disable-availability-checking -enable-bare-slash-regex > %t/cpp-parser.ast.raw
-
-// Filter out any addresses in the dump, since they can differ.
-// RUN: sed -E 's#0x[0-9a-fA-F]+##g' %t/cpp-parser.ast.raw > %t/cpp-parser.ast
-// RUN: sed -E 's#0x[0-9a-fA-F]+##g' %t/astgen.ast.raw > %t/astgen.ast
-
-// RUN: %diff -u %t/astgen.ast %t/cpp-parser.ast
-
```

Comment:
SwiftParser/ASTGen has better recovery so AST might be different in invalid cases. It's not worth to emit the same AST. 
Specifically, e.g. C++  parser discards the whole attribute for `@_silgen_name("whatever", extra)` because of the `extra` argument. But ASTGen recovers it.